plugins/Diagnostics/Diagnostic/SystemFormatCheck.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+namespace Piwik\Plugins\Diagnostics\Diagnostic;
+
+use Piwik\Config;
+use Piwik\SettingsServer;
+use Piwik\Translation\Translator;
+
+/**
+ * Check that the memory limit is enough.
+ */
+class SystemFormatCheck implements Diagnostic
```

Comment:
Might be good to rename this class and file to something more meaningful. What about `PHPBinaryCheck`? Also the comment above is incorrect.